src/test/kotlin/com/igorwojda/integer/generateallpairs/challenge.kt

```diff
@@ -4,7 +4,13 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getAllPairs(n: Int): List<Pair<Int, Int>> {
-    TODO("not implemented")
+    val pairs= ArrayList<Pair<Int,Int>>()
```

Comment:
Instead of directly using ArrayList, please use a more generic type of collection: https://kotlinlang.org/docs/collections-overview.html#collection-types